scripts/exglobal_pmgr.sh.ecf

```diff
@@ -45,7 +45,7 @@ do
   for fhr in $postjobs
   do 
     fhr3=`printf "%03d" $fhr`   
-    if [ -s ${COMIN}/${RUN}.${cycle}.logf${fhr}.nemsio -o  -s ${COMIN}/${RUN}.${cycle}.logf${fhr3}.nemsio ]
+    if [ -s ${COMIN}/${RUN}.${cycle}.logf${fhr}.txt -o  -s ${COMIN}/${RUN}.${cycle}.logf${fhr3}.txt ]
```

Comment:
all files should be in logf${fhr3} format.  Is the first part logf${fhr} kept for backward compatibility?

Reply:
This change is targeting operational GFS V16. Since GFS V16 is not outputting nemsio format, the backward compatibility might not be necessary.